### PR TITLE
Eduroam documentation

### PR DESCRIPTION
**Pull request by lzammit**
# Description
Eduroam documentation

# Impacts
Documentation  file

# Delete branch after merge
YES

# Checklist
- [X] Document the feature

# NEWS file entries
Eduroam documentation

### CODE REVIEW

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 1]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+Understanding of the eduroam authentication workflow.
+
+Local authentication
+^^^^^^^^^^^^^^^^^^^^
+
+image::internal-eduroam-authentication.png[scaledwidth="100%",alt="Internal eduroam authentication"]
+
+1: The device connects on the eduroam SSID.
+
+2: The access point forward the authentication request to the wireless controller.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: forwards

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 2]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+3: The controller sends the radius authentication to PacketFence on port 11812.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: RADIUS authentication request

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 3]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+4: PacketFence checks if it's a local REALM.
+
+5: If it's local REALM, PacketFence does a NTLM request to the AD to verify the identity.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: to the Active Directory (AD) domain controller

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 4]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+5a: The AD validated the credentials.
+
+5b: The AD did not validate the credentials. PacketFence sends a Radius Reject
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: a RADIUS access-reject response

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 5]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+4: PacketFence checks if it's a local REALM.
+
+5: If it's local REALM, PacketFence does a NTLM request to the AD to verify the identity.
+
+5a: The AD validated the credentials.
+
+5b: The AD did not validate the credentials. PacketFence sends a Radius Reject 
+
+5c: After a successful NTLM authentication, PacketFence returns a Radius Acess Accept to the wireless controller to apply the production VLAN for that mac address.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: RADIUS access-accept response

Comment (extrafu) [reply docs/PacketFence_Installation_Guide.asciidoc]: mac -> MAC

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 6]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+6: If it's a not local REALM, PacketFence proxy the radius request to the eduroam servers
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: proxies the RADIUS

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 7]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+7: The eduroam servers validate the identity
+
+8: PacketFence returns a Radius Acess Accept to the wireless controller to apply the production VLAN for that mac address.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: RADIUS access-accept response

Comment (extrafu) [reply docs/PacketFence_Installation_Guide.asciidoc]: mac -> MAC

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 8]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+Configure the Eduroam source
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Go to the PacketFence administration web interface and go to *Configuration -> Policies and Access Control -> Authentication Sources*.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: Open the PacketFence administration...

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 9]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+Local *Exclusive Sources* and click on 'New excluse source' then 'Eduroam'.
+
+The information to setup that source could be found on the eduroam platform.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: to setup -> to configure

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 10]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+Create the Connection Profile for local authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Go to *Configuration -> Policies and Access Control -> Connection Profiles -> New Connection Profile*
+
+Create the Connection Profile named 'Local and external eduroam authentication' Check "Automatically register devices" then create a SSID filter 'eduroam'. Make sure to add the Active Directory source to match on the local users.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: Create a connection profile named

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 11]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+
+Inbound authentication (TLRS to PF)
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: PF -> PacketFence

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 12]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+image::inbound-eduroam-authentication.png[scaledwidth="100%",alt="Inbound eduroam authentication"]
+
+1: Eduroam sends the radius authentication to a public IP address (NAT/PAT) bound to PacketFence on the management IP address (Management VIP for a cluster) on port 1812.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: RADIUS authentication request

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 13]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+2: PacketFence forwards the NTLM request to the Active Directory.
+
+3a: User identity successfuly authentication on the AD.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: Successful user identify authentication on the AD

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 14]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+3b: NTLM request fails because of a bad identity
+
+4: PacketFence replies to the Eduroam servers either a RADIUS Access Accept for a sucessful authentication or a RADIUS Access Reject for an unsuccessful authentication. PacketFence sets the REALM to eduroam for any sucessful authentication.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: eduroam.... RADIUS access-accept .... RADIUS access-reject... for all successful authentications.

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 15]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+First, you need to refer to the previous step 'Configure the Eduroam source'.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: eduroam

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 16]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+Once the source is configured, you will need to create a new Connection profile.
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: connection

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 17]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+Create the Connection Profile for outbound authentication
```
Comment (extrafu) [inline docs/PacketFence_Installation_Guide.asciidoc]: connection profile for outbound authentications

**Comment by nqb** [comment]
@lzammit, please follow AsciiDoc conventions https://packetfence.org/doc/PacketFence_Developers_Guide.html#_documentation_conventions

**Comment by lzammit** [comment]
@nqb if you are referencing to the list, it does not work for me because of the 2 levels, thanks.

**Comment by nqb** [comment]
Hello @lzammit,

AsciiDoc supports several level of lists, take a look at my previous link there is some examples.

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 18]
```diff
@@ -1220,7 +1220,77 @@ _____________________
 
 PacketFence supports Eduroam and allows participating institutions to authenticate both locally visiting users from other institutions as well as allowing other institutions to authenticate local users.
 
-In order for PacketFence to allow Eduroam authentication, you need to configure an exclusive Eduroam source in Configuration->Authentication Sources and restart the RADIUS and IPTables services.
+Understanding of the eduroam authentication workflow.
+
+Local authentication
+^^^^^^^^^^^^^^^^^^^^
```
Comment (nqb) [inline docs/PacketFence_Installation_Guide.asciidoc]: You can use following syntax: `===`, see https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#section-titles

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 19]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+5c: After a successful NTLM authentication, PacketFence returns a Radius Acess Accept to the wireless controller to apply the production VLAN for that MAC address.
+
+6: If it's a not local REALM, PacketFence proxies the radius request to the eduroam servers
+
+7: The eduroam servers validate the identity
+
+8: PacketFence returns a Radius Acess Accept to the wireless controller to apply the production VLAN for that MAC address.
+
+Configure the Eduroam source
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (nqb) [inline docs/PacketFence_Installation_Guide.asciidoc]: You can use following syntax: `===`, see https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#section-titles

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 20]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+Open the PacketFence administration web interface and go to *Configuration -> Policies and Access Control -> Authentication Sources*.
+
+Local *Exclusive Sources* and click on `New excluse source` then `Eduroam`.
+
+The information to configure that source could be found on the eduroam platform.
+
+Create the connection profile for local authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (nqb) [inline docs/PacketFence_Installation_Guide.asciidoc]: You can use following syntax: `===`, see https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#section-titles

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 21]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+
+Go to *Configuration -> Policies and Access Control -> Connection Profiles -> New Connection Profile*
+
+Create a connection profile named `Local and external eduroam authentication` Check "Automatically register devices" then create a SSID filter `eduroam`. Make sure to add the Active Directory source to match on the local users.
+
+
+Inbound authentication (TLRS to PF)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (nqb) [inline docs/PacketFence_Installation_Guide.asciidoc]: You can use following syntax: `===`, see https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#section-titles

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 22]
```diff
@@ -1220,7 +1220,77 @@ _____________________
...
+3b: NTLM request fails because of a bad identity
+
+4: PacketFence replies to the Eduroam servers either a RADIUS Access Accept for a sucessful authentication or a RADIUS access reject for an unsuccessful authentication. PacketFence sets the REALM to eduroam for all successful authentications.
+
+First, you need to refer to the previous step `Configure the Eduroam source`.
+
+Once the source is configured, you will need to create a new connection profile.
+
+Create the connection profile for outbound authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (nqb) [inline docs/PacketFence_Installation_Guide.asciidoc]: You can use following syntax: `===`, see https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#section-titles

**Review by nqb — CHANGES_REQUESTED** [review]
For path to menus, you can use following syntax: `_Configuration -> Policies and Access Control_` as mentioned in conventions too.

**Review by extrafu — COMMENTED** [review]
You pervert.

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 23]
```diff
@@ -1220,7 +1220,72 @@ _____________________
...
+
+5b: The AD did not validate the credentials. PacketFence sends a RADIUS Reject 
+
+5c: After a successful NTLM authentication, PacketFence returns a Radius Acess Accept to the wireless controller to apply the production VLAN for that MAC address.
+
+6: If it's a not local REALM, PacketFence proxies the radius request to the eduroam servers
+
+7: The eduroam servers validate the identity
+
+8: PacketFence returns a Radius Acess Accept to the wireless controller to apply the production VLAN for that MAC address.
```
Comment (fdurand) [inline docs/PacketFence_Installation_Guide.asciidoc]: Access Accept

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 24]
```diff
@@ -1220,7 +1220,72 @@ _____________________
...
+
+4: PacketFence checks if it's a local REALM.
+
+5: If it's local REALM, PacketFence does a NTLM request to the Active Directory (AD) domain controller to verify the identity.
+
+5a: The AD validated the credentials.
+
+5b: The AD did not validate the credentials. PacketFence sends a RADIUS Reject 
+
+5c: After a successful NTLM authentication, PacketFence returns a Radius Acess Accept to the wireless controller to apply the production VLAN for that MAC address.
```
Comment (fdurand) [inline docs/PacketFence_Installation_Guide.asciidoc]: Access Accept

**Comment by nqb** [comment]
@lzammit, list items need to be adjust.

**Comment by lzammit** [comment]
Done!

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 25]
```diff
@@ -1220,7 +1220,62 @@ _____________________
...
 PacketFence supports Eduroam and allows participating institutions to authenticate both locally visiting users from other institutions as well as allowing other institutions to authenticate local users.
 
-In order for PacketFence to allow Eduroam authentication, you need to configure an exclusive Eduroam source in Configuration->Authentication Sources and restart the RADIUS and IPTables services.
+Understanding of the eduroam authentication workflow.
+
+==== Local authentication
+
+image::internal-eduroam-authentication.png[scaledwidth="100%",alt="Internal eduroam authentication"]
+
+* 1: The device connects on the eduroam SSID.
```
Comment (nqb) [inline docs/PacketFence_Installation_Guide.asciidoc]: If you want to use ordered lists, you should use followin syntax:
```
. First item
. Second item
. Third item
```

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 26]
```diff
@@ -1220,7 +1220,62 @@ _____________________
...
+* 5c: After a successful NTLM authentication, PacketFence returns a Radius Access Accept to the wireless controller to apply the production VLAN for that MAC address.
+* 6: If it's a not local REALM, PacketFence proxies the radius request to the eduroam servers
+* 7: The eduroam servers validate the identity
+* 8: PacketFence returns a Radius Access Accept to the wireless controller to apply the production VLAN for that MAC address.
+
+==== Configure the Eduroam source
+
+Open the PacketFence administration web interface and go to _Configuration -> Policies and Access Control -> Authentication Sources_.
+
+Local *Exclusive Sources* and click on `New excluse source` then `Eduroam`.
```
Comment (nqb) [inline docs/PacketFence_Installation_Guide.asciidoc]: I don't understand this sentence.

There is a typo too : exclus**i**ve in place of excluse

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 27]
```diff
@@ -1220,7 +1220,62 @@ _____________________
...
+Go to _Configuration -> Policies and Access Control -> Connection Profiles -> New Connection Profile_
+
+Create a connection profile named `Local and external eduroam authentication` Check `Automatically register devices` then create a SSID filter `eduroam`. Make sure to add the Active Directory source to match on the local users.
+
+
+==== Inbound authentication (TLRS to PF)
+
+image::inbound-eduroam-authentication.png[scaledwidth="100%",alt="Inbound eduroam authentication"]
+
+1: Eduroam sends the RADIUS authentication to a public IP address (NAT/PAT) bound to PacketFence on the management IP address (Management VIP for a cluster) on port 1812.
```
Comment (nqb) [inline docs/PacketFence_Installation_Guide.asciidoc]: If you want to use ordered lists, you should use followin syntax:
```
. First item
. Second item
. Third item
```

**Comment by julsemaan** [comment]
But the main question is: Are the numbered lists following our standards?

**Comment by extrafu** [comment]
@julsemaan I gave up on that, @lzammit can't even read our own guidelines.

**Pull request merged**